### ISSUE TITLE
victoria-metrics-single helm chart 0.10.1 statefulset breaks when providing image pull secrets

### ISSUE THREAD

**Issue by nick-oconnor** [VictoriaMetrics/helm-charts#1285]
I get the following error attempting to deploy:

```
Error: YAML parse error on victoria-metrics-single/templates/server-statefulset.yaml: error converting YAML to JSON: yaml: line 31: did not find expected ',' or ']'
```

The chart applies cleanly when I omit:
```
global:
  imagePullSecrets:
    - name: my-secret
```
I suspect that the following line is expecting a string instead of an array:
https://github.com/VictoriaMetrics/helm-charts/blob/c6b08e553712124165d5cb20a6776b28752dc058/charts/victoria-metrics-single/templates/server-statefulset.yaml#L40

**Comment by AndrewChubatiuk** [comment]
hey @nick-oconnor 
thanks for submitting an issue, create PR with a fix

**Comment by nick-oconnor** [comment]
That was super fast! :pray: :rocket:

**Comment by AndrewChubatiuk** [comment]
global image pull secrets are available starting [0.11.0](https://github.com/VictoriaMetrics/helm-charts/releases/tag/victoria-metrics-single-0.11.0)
